<Type Name="FunctionInputConverterException" FullName="Microsoft.Azure.Functions.Worker.FunctionInputConverterException">
  <TypeSignature Language="C#" Value="public class FunctionInputConverterException : Microsoft.Azure.Functions.Worker.FunctionWorkerException" />
  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit FunctionInputConverterException extends Microsoft.Azure.Functions.Worker.FunctionWorkerException" />
  <TypeSignature Language="DocId" Value="T:Microsoft.Azure.Functions.Worker.FunctionInputConverterException" />
  <TypeSignature Language="VB.NET" Value="Public Class FunctionInputConverterException&#xA;Inherits FunctionWorkerException" />
  <TypeSignature Language="F#" Value="type FunctionInputConverterException = class&#xA;    inherit FunctionWorkerException" />
  <AssemblyInfo>
    <AssemblyName>Microsoft.Azure.Functions.Worker.Core</AssemblyName>
    <AssemblyVersion>1.6.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>Microsoft.Azure.Functions.Worker.FunctionWorkerException</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
            The exception that is thrown when input conversion for the function invocation fails.
            </summary>
    <remarks>To be added.</remarks>
  </Docs>
  <Members>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FunctionInputConverterException (string? message);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message) cil managed" />
      <MemberSignature Language="DocId" Value="M:Microsoft.Azure.Functions.Worker.FunctionInputConverterException.#ctor(System.String)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String)" />
      <MemberSignature Language="F#" Value="new Microsoft.Azure.Functions.Worker.FunctionInputConverterException : string -&gt; Microsoft.Azure.Functions.Worker.FunctionInputConverterException" Usage="new Microsoft.Azure.Functions.Worker.FunctionInputConverterException message" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>Microsoft.Azure.Functions.Worker.Core</AssemblyName>
        <AssemblyVersion>1.6.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
      </Parameters>
      <Docs>
        <param name="message">The message that describes the error.</param>
        <summary>
            Initializes a new instance of the FunctionInputConverterException class with a specified error message.
            </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
    <Member MemberName=".ctor">
      <MemberSignature Language="C#" Value="public FunctionInputConverterException (string? message, Exception? innerException);" />
      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(string message, class System.Exception innerException) cil managed" />
      <MemberSignature Language="DocId" Value="M:Microsoft.Azure.Functions.Worker.FunctionInputConverterException.#ctor(System.String,System.Exception)" />
      <MemberSignature Language="VB.NET" Value="Public Sub New (message As String, innerException As Exception)" />
      <MemberSignature Language="F#" Value="new Microsoft.Azure.Functions.Worker.FunctionInputConverterException : string * Exception -&gt; Microsoft.Azure.Functions.Worker.FunctionInputConverterException" Usage="new Microsoft.Azure.Functions.Worker.FunctionInputConverterException (message, innerException)" />
      <MemberType>Constructor</MemberType>
      <AssemblyInfo>
        <AssemblyName>Microsoft.Azure.Functions.Worker.Core</AssemblyName>
        <AssemblyVersion>1.6.0.0</AssemblyVersion>
      </AssemblyInfo>
      <Parameters>
        <Parameter Name="message" Type="System.String" />
        <Parameter Name="innerException" Type="System.Exception" />
      </Parameters>
      <Docs>
        <param name="message">The message that describes the error.</param>
        <param name="innerException">The exception that is the cause of the current exception
            or a null reference if no inner exception is specified..</param>
        <summary>
            Initializes a new instance of the FunctionInputConverterException class with a specified error message
            and a reference to the inner exception that is thecause of this exception.
            </summary>
        <remarks>To be added.</remarks>
      </Docs>
    </Member>
  </Members>
</Type>
